src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,170 @@
+import domain.result.PlayerProfitStatistics;
+import domain.user.*;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toList;
+import static view.OutputView.*;
+
+public class BlackJackGame {
+    private static final int FIRST = 0;
+
+    public static void main(String[] args) {
+        Players players = inputPlayers();
+        Dealer dealer = new Dealer();
+        initializeCardsOfAllUsers(players, dealer, new RandomCardShuffler());
+        showCardsOfAllUsers(players, dealer);
+        askWantMoreCardToAllPlayers(players);
+        distributeCardToDealer(dealer);
+        showDealerAndPlayersResult(players, dealer);
+        showDealerAndPlayersProfit(players, dealer);
+    }
+
+    private static Players inputPlayers() {
+        List<PlayerName> playerNames = inputPlayerNames();
+        List<Player> players = playerNames.stream()
+                .map(BlackJackGame::convertNamesAndBettingMoneyIntoPlayers)
+                .collect(toList());
+        return new Players(players);
+    }
+
+    private static List<PlayerName> inputPlayerNames() {
+        OutputView.printPleaseInputNames();
+        return convertInputNamesIntoPlayerNames(InputView.inputNames());
+    }
+
+    private static List<PlayerName> convertInputNamesIntoPlayerNames(List<String> playerNames) {
+        return playerNames.stream()
+                .map(String::trim)
+                .map(PlayerName::new)
+                .collect(toList());
+    }
+
+    private static Player convertNamesAndBettingMoneyIntoPlayers(PlayerName playerName) {
+        OutputView.printAskHowMuchBettingToPlayer(playerName);
+        BettingMoney bettingMoney = convertAmountIntoBettingMoney(InputView.inputAmount());
+        return new Player(playerName, bettingMoney);
+    }
+
+    private static BettingMoney convertAmountIntoBettingMoney(BigDecimal amount) {
+        return new BettingMoney(amount);
+    }
+
+    private static void initializeCardsOfAllUsers(Players players, Dealer dealer, CardShuffler cardShuffler) {
+        Deck.initializeDeck(cardShuffler);
+        Deck.distributeCardsToPlayersAndDealer(players, dealer);
+        OutputView.printHandedOutTwoCardsToPlayers(players);
+    }
+
+    private static void showCardsOfAllUsers(Players players, Dealer dealer) {
+        showCardsOfDealer(dealer);
+        showCardsOfPlayers(players);
+    }
+
+    private static void showCardsOfDealer(Dealer dealer) {
+        Card firstCard = dealer.getCards().get(FIRST);
+        String firstCardTypeName = firstCard.getType().getTypeName();
+        String firstCardSymbolName = firstCard.getSymbol().getSymbolName();
+        OutputView.printCardsHeldByDealer(firstCardTypeName, firstCardSymbolName);
+    }
+
+    private static void showCardsOfPlayers(Players players) {
+        for (Player player : players.getPlayers()) {
```

Comment:
```suggestion
        List<Player> players = players.getPlayers();
        for (Player player : players) {
```
위와 같이 변경하는게 더 좋은건 이전 피드백을 통해서 알고 계실 것 같습니다 😉

Reply:
이 부분에 관하여 [참고자료](https://namocom.tistory.com/754)를 찾아봤습니다!